### PR TITLE
Streamline JS README

### PR DESCRIPTION
**Pull request by SuaYoo**
Streamlines the @telnyx/webrtc README by cross-linking with examples in docs, to reduce the amount of duplicated (and erroneous) example code.

Side by side comparison: [Current README](https://github.com/team-telnyx/webrtc/blob/main/packages/js/README.md) | [Proposed README](https://github.com/team-telnyx/webrtc/blob/Update_js_readme/packages/js/README.md)

### CODE REVIEW

File: packages/js/README.md [hunk 1]
```diff
@@ -37,199 +29,155 @@ import { TelnyxRTC } from '@telnyx/webrtc';
...
+    if (notification.type === 'callUpdate') {
+      activeCall = notification.call;
     }
   });
-
-// Connect and login
-client.connect();
-
-// You can disconnect when you're done
-// client.disconnect();
```
Comment (DeividVeloso) [inline packages/js/README.md:119]: I think we should keep this example, this is easier to customer see what he can do with our SDK.

@hugobdev any thoughts?

File: packages/js/README.md [hunk 2]
```diff
@@ -37,199 +29,155 @@ import { TelnyxRTC } from '@telnyx/webrtc';
...
-      autoplay="true"
-      playsinline="true"
-    />
+// Send digits and keypresses
+call.dtmf('1234');
 
+// Call states that can be toggled
+call.hold();
+call.muteAudio();
 ```
```
Comment (DeividVeloso) [inline packages/js/README.md:144]: What do yu think to keep the main methods here, for example:

```
call.answer()
call.newCall()
....
```

**Review by DeividVeloso — CHANGES_REQUESTED** [review]
I put some MINOR comments

File: packages/js/README.md (hunk 1 above)
Comment (hugobdev) [reply packages/js/README.md:119]: I think that we can keep a simpler readme as Sua proposed, and guide the user to see the full-blown example as they are actually integrating with the SDK.

File: packages/js/README.md [hunk 3]
```diff
@@ -37,199 +29,155 @@ import { TelnyxRTC } from '@telnyx/webrtc';
...
-// Answer an incoming call
-call.answer();
-// Hangup (reject) an incoming call
-call.hangup();
+// And in your HTML, replace the audio element with video.
+//  <video id="remoteMedia" autoplay="true" playsinline="true" />
 ```
 
----
+> See [TelnyxRTC.newCall](./docs/ts/classes/telnyxrtc.md#newCall) for all options.
```
Comment (hugobdev) [inline packages/js/README.md:117]: It will be great when we can link to the final dev docs reference.

File: packages/js/README.md (hunk 2 above)
Comment (hugobdev) [reply packages/js/README.md:144]: `newCall` and `answer` were just used at the top. I think it is fine removing them from this last example.

File: packages/js/README.md [hunk 4]
```diff
@@ -37,199 +29,155 @@ import { TelnyxRTC } from '@telnyx/webrtc';
...
 
-We've included a few [examples in Javascript(ES6)](https://github.com/team-telnyx/webrtc/tree/main/packages/js/examples/vanilla) to help you get started.
+We've included a few [examples in Javascript (ES6)](https://github.com/team-telnyx/webrtc/tree/main/packages/js/examples/vanilla) to help you get started.
 
-```
-cd examples/vanilla
-open index.html
+```sh
+$ cd examples/vanilla
+$ open index.html
```
Comment (hugobdev) [inline packages/js/README.md:157]:
```suggestion
cd examples/vanilla
open index.html
```

Removing the `$` makes it easier to copy this example code

File: packages/js/README.md [hunk 5]
```diff
@@ -37,199 +29,155 @@ import { TelnyxRTC } from '@telnyx/webrtc';
...
-```
-cd examples/react-audio
-npm run setup
-npm install
-npm run storybook
+```sh
+$ cd examples/react-audio
+$ npm run setup
+$ npm install
+$ npm run storybook
```
Comment (hugobdev) [inline packages/js/README.md:180]:
```suggestion
$ cd examples/react-audio
$ npm run setup
$ npm install
$ npm run storybook
```
```suggestion
cd examples/react-audio
npm run setup
npm install
npm run storybook
```

File: packages/js/README.md [hunk 6]
```diff
@@ -239,17 +187,21 @@ Screenshot:
...
-```
-cd examples/react-video
-npm run setup
-npm install
-npm start
+```sh
+$ cd examples/react-video
+$ npm run setup
+$ npm install
+$ npm start
```
Comment (hugobdev) [inline packages/js/README.md:194]:
```suggestion
cd examples/react-video
npm run setup
npm install
npm start
```

Comment (hugobdev) [reply packages/js/README.md:194]: If you intended to keep the `$` to make it clear that those are separate commands, you can add `;` at the end of each line

File: packages/js/src/TelnyxRTC.ts [hunk 7]
```diff
@@ -31,7 +31,14 @@ export default class TelnyxRTC extends TelnyxRTCClient {
...
    *
-   * Setting `ringtoneFile` and `ringbackFile`:
+   * ### Custom ringtone and ringback
+   *
+   * Custom ringback and ringtone files can be a wav/mp3 in your local public folder
+   * or a file hosted on a CDN, ex: https://cdn.company.com/sounds/call.mp3.
+   *
+   * To use the `ringbackFile`, make sure the "Generate Ringback Tone" option is **disabled**
+   * in your [Telnyx Portal connection](https://portaldev.telnyx.com/#/app/connections)
+   * configuration (Inbound tab.)
```
Comment (hugobdev) [inline packages/js/src/TelnyxRTC.ts:41]: 👍

**Review by hugobdev — CHANGES_REQUESTED** [review]
Few comments, nothing major. Great job making this simpler

**Pull request merged**